cmd/metal-api/internal/service/v1/switch.go

```diff
@@ -54,12 +55,19 @@ type SwitchRegisterRequest struct {
 	SwitchBase
 }
 
+type SwitchNotifyRequest struct {
+	Duration time.Duration `json:"sync_duration" description:"the duration of the switch synchronization"`
```

Comment:
Good idea to have the sync duration stored, but how can we access it ? Switchresponse does not contain it.
Maybe late we can report this in the metrics ?

Reply:
We can report this as a metric later but the SwitchResponse was enriched with additional fields as well.

Reply:
sure but not he duration, as  far as i can see